.clj-kondo/hooks/common.clj

```diff
@@ -46,9 +46,10 @@
   ;; unused variables since Kondo was smart enough to realize that we sometimes were calling pure functions early in
   ;; the `do` and the value was getting thrown away. `print` tricks Kondo into thinking everything is used.
   [{{[_ & args] :children} :node}]
```

Comment:
Is this (i.e. `(constantly nil)`) a legit substitute for `print` in these cases? I'm thinking so, but wanted a second opinion.

Reply:
I think that substitute works ok ~but maybe the log/info token would work better.~

Reply:
Putting `print` in was my hack from a couple weeks ago, and replacing it with `(constantly nil)` is definitely clearer and has my enthusiastic endorsement. If the Kondo check passes it's a legit substitute, I don't think there's anything subtle to consider.

Reply:
(that said, please update the comment too :) )

Reply:
Done!